{"name": "Изменение данных авторизованного пользователя", "status": "broken", "statusDetails": {"message": "KeyError: 'accessToken'", "trace": "self = <tests.test_udate_user_data.TestUpdateUserData object at 0x109e60350>, register_new_user_return_response = <Response [403]>\nchanged_data = '{\"email\": data[0], \"password\":data[1], \"name\":fake.name()}'\n\n    @allure.title('Изменение данных авторизованного пользователя')\n    @pytest.mark.parametrize('changed_data', ['{\"email\": fake.email(), \"password\": data[1],\"name\": data[2]}',\n                                              '{\"email\": data[0], \"password\": fake.password(), \"name\": data[2]}',\n                                              '{\"email\": data[0], \"password\":data[1], \"name\":fake.name()}'],\n                             ids=['email', 'password', 'name'])\n    def test_update_authorized_user(self, register_new_user_return_response, changed_data):\n        data = register_new_user_return_response\n        fake = Faker()\n        payload = changed_data\n>       access_token = data.json()['accessToken']\nE       KeyError: 'accessToken'\n\ntests/test_udate_user_data.py:21: KeyError"}, "parameters": [{"name": "changed_data", "value": "'{\"email\": data[0], \"password\":data[1], \"name\":fake.name()}'"}], "start": 1721244087566, "stop": 1721244087857, "uuid": "0044f6db-3cc8-49c6-8610-ca7a2f49a083", "historyId": "6a4b905dc8abbceca68e3589f7d32215", "testCaseId": "60625cfcd58b41d4e85909c732e93f2c", "fullName": "tests.test_udate_user_data.TestUpdateUserData#test_update_authorized_user", "labels": [{"name": "suite", "value": "Изменение данных пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestUpdateUserData"}, {"name": "host", "value": "NB-ISC-14346"}, {"name": "thread", "value": "40900-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_udate_user_data"}]}